ui/src/useRoom.ts

```diff
@@ -144,9 +145,10 @@ const clientSession = async ({
         }
     };
     peer.ontrack = (event) => {
+        const kind = event.track.kind;
         const stream = new MediaStream();
         stream.addTrack(event.track);
-        onTrack(stream);
+        onTrack(stream, kind);
```

Comment:
Initially, I considered implementing it that way as well. However, with the current track handler, a new MediaStream is created upon receiving, so in order to include multiple tracks in a single MediaStream, we would need to change the position where the MediaStream is created and move it to a higher scope.

Additionally, MediaStream is merely a view of the underlying WebRTC tracks, and I didn't think that having more of them would cause significant performance issues. However, I might be mistaken on this point.

I implemented it this way to keep the changes small, but if it's necessary to combine them into one, I will adjust accordingly.